README.md

```diff
@@ -39,6 +41,16 @@ Maven users would add to their `pom.xml`
 
 A full example of a Java API client can be found [here](./examples/java/printing-client).
 
+### Python
+
+You can install precompiled client stubs for Python with `pip` command.
```

Comment:
May as well be consistent in our wording.

`We provide precompiled client stubs for Python. Python users can install them with `pip`.`